staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -269,6 +271,10 @@ func (o *MirrorOptions) Run() error {
 		}
 	}
 
+	var currentRepo string
+	exists := struct{}{}
+	repoMap := map[string]struct{}{}
```

Comment:
style: always use `repositories := make(map[string]struct{}{})` to init variables, and don't use `Map` in variables (map is implicit in the name usually)